content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -908,6 +908,8 @@ subjects:
   name: user-1
 ```
 
+Note that - as with any RBAC verb - you may omit `resourceNames` to allow `user-1` to grant other users _any_ ClusterRole in the namespace `user-1-namespace`.
```

Comment:
```suggestion
Note that—as with any RBAC verb—you may omit the `resourceNames` field to allow `user-1` to grant other users _any_ `ClusterRole` in the namespace `user-1-namespace`.
```
Note that the typesetting for kind names like "ClusterRole" is inconsistent in this document. Sometimes it's plain text, and other times it's set in monospace, as on line 866 above.

Reply:
could also inline this as a comment in the example role:

```yaml
  # omit resourceNames to allow binding any clusterrole
  resourceNames: ["admin","edit","view"]
```

Reply:
I prefer the latter approach here - done.